include/nil/actor/zk/snark/routing/as_waksman.hpp

```diff
@@ -164,7 +164,7 @@ namespace nil {
                  *
                  * If top = true, return the top wire, otherwise return bottom wire.
                  */
-                std::size_t as_waksman_switch_output(size_t num_packets, std::size_t row_offset, std::size_t row_idx,
+                inline std::size_t as_waksman_switch_output(size_t num_packets, std::size_t row_offset, std::size_t row_idx,
                                                      bool use_top) {
                     std::size_t relpos = row_idx - row_offset;
                     assert(relpos % 2 == 0 && relpos + 1 < num_packets);
```

Comment:
Maybe let's use assert with some message explanation?

Reply:
Will change this later.